readme.md

```diff
@@ -72,5 +292,4 @@ See the [types](source/index.d.ts) for now.
 
 ## Related
 
-- [execa](https://github.com/sindresorhus/execa) - Process execution for humans
 - [unicorn-magic](https://github.com/sindresorhus/unicorn-magic/blob/6614e1e82a19f41d7cc8f04df7c90a4dfe781741/node.d.ts#L77-L125) - Slightly improved `child_process#execFile`
```

Comment:
It seems like `unicorn-magic` provides with a shortcut for `util.promisify(execFile)`. However, it is not documented and seems to be mostly used for your own packages? Should it be mentioned here then?